src/patch-events.js

```diff
@@ -266,11 +281,7 @@ export function addEventListener(type, fn, optionsOrCapture) {
   if (fn.__eventWrappers) {
     // Stop if the wrapper function has already been created.
     for (let i = 0; i < fn.__eventWrappers.length; i++) {
-      if (fn.__eventWrappers[i].node === this &&
```

Comment:
I must find the PR where I measured this. It was showing that, because this is a high frequency path, invoking functions had a cost